### PR TITLE
Added Models and Views

### PR DESCRIPTION
**Pull request by pwaaron**

### CODE REVIEW

File: flask-part/api/models.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	name = db.Column(db.String(60))
+	phone = db.Column(db.Integer)
+	address = db.Column(db.String(60))
+
+class Menu(db.Model):
+	menuid = db.Column(db.Integer, primary_key = True)
+	restaurantid = db.Column(db.Integer)
+	name = db.Column(db.String(60))
+	price = db.Column(db.Float)
+
```
Comment (agnesnatasya) [inline flask-part/api/models.py:25]: Restaurant, Menu and Item classes are unnecessary, please delete

File: flask-part/api/models.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	orderid = db.Column(db.Integer, primary_key = True)
+	restaurantid = db.Column(db.Integer)
+	deadline_time = db.Column(db.Date)
+	shipping_address = db.Column(db.String(120))
+
+class Item(db.Model):
+	itemid = db.Column(db.Integer, primary_key = True)
+	menuid = db.Column(db.Integer)
+	restaurantid = db.Column(db.Integer)
+	userid = db.Column(db.Date)
```
Comment (agnesnatasya) [inline flask-part/api/models.py:36]: Item class is unnecessary, please delete

**Comment by pwaaron** [comment]
@agnesnatasya please review

File: flask-part/api/views.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	return 'YAS', 201
+
+@main.route('/neworder', methods = ['POST'])
+def neworder():
+	return 'YAS', 201
+
+@main.route('/myorder')
+def neworder():
+	return 'YAS'
+"""
```
Comment (agnesnatasya) [inline flask-part/api/views.py:33]: Please you change the name of the routing

**Comment by agnesnatasya** [comment]
Looks good to merge!

**Pull request merged**